workspace/apps/pidp/src/app/features/access/pages/hcim-reenrolment/hcim-reenrolment.page.html

```diff
@@ -113,45 +139,40 @@
 
         <div class="row">
           <div class="col-12">
-            <ui-form-section [showAction]="true">
+            <ui-form-section [showAction]="false">
               <mat-form-field class="w-100">
                 <input
                   matInput
-                  placeholder="HealthNetBC Username"
+                  placeholder="Username"
                   formControlName="ldapUsername" />
                 <mat-error>Required</mat-error>
               </mat-form-field>
-              <ng-container uiContextHelpContent>
-                <strong>Username Format:</strong> 1767-bsmith (typical format of
-                username: Organization ID-First Initial, Last Name)
-              </ng-container>
             </ui-form-section>
           </div>
+          <ui-page-section-subheader heading="HCIMWeb HealthNetBC Password">
```

Comment:
This is a common mistake using grids.  You shouldn't place any markup between the opening and closing `.row` element unless it's an element with a `.col` (or equivalent `.col-6`, etc).  Rows shouldn't have non-column elements as direct children you can get some odd behaviours doing this on different viewports depending on the situation.